src/main/java/com/google/sps/data/ScheduledInterviewRequest.java

```diff
@@ -78,6 +90,8 @@ public String toString() {
         "interviewee",
         interviewee,
         "role",
-        role);
+        role,
+        "hasStarted",
```

Comment:
Made this comment in another PR.

Does it make sense to inline the field names into the format string? I don't see any good reason to substitute them in.

Reply:
Done